packages/runtime/container-runtime/src/opLifecycle/opGroupingManager.ts

```diff
@@ -71,9 +104,13 @@ export class OpGroupingManager {
 			});
 		}
 
+		let groupedBatchId;
```

Comment:
Maybe put a comment saying "We expect this will be on the first message, if present at all."